rpcs3/Emu/Cell/Modules/sceNp2.cpp

```diff
@@ -611,6 +611,11 @@ error_code sceNpMatching2SignalingGetConnectionInfo(
 		return SCE_NP_MATCHING2_ERROR_NOT_INITIALIZED;
 	}
 
+	if (!connInfo || code < 1 || code > 6)
```

Comment:
code is actually already checked in the switch below.
You just need to return the SCE_NP_MATCHING2_ERROR_INVALID_ARGUMENT instead of logging a fatal